packages/rpc-graphql/src/schema/type-defs/account.ts

```diff
@@ -57,8 +57,8 @@ export const accountTypeDefs = /* GraphQL */ `
         rentEpoch: Epoch
         addresses: [Address]
         authority: Account
-        deactivationSlot: Slot
-        lastExtendedSlot: Slot
+        deactivationSlot: Block
+        lastExtendedSlot: Block
```

Comment:
Still think these should be renamed to reflect the new return type.

Reply:
I truly waffled on this a bit actually. I just wanted to be careful about straying too far from the defined data, such as an account whose fields are "something-slot" and now they have to call it "block".

I couldn't decide if this was acceptable, really.

Reply:
If we leave it this way, it's going to lead to a lot of aggravating code like:

```ts
if (data.slot.slot < 123n) {
    // ...
}
```